src/edfi-performance-test/edfi_performance_test/api/client/ed_fi_basic_api_client.py

```diff
@@ -74,6 +80,24 @@ def get_headers(self):
             "Content-Type": "application/json",
         }
 
+    def _post(self, endpoint: str, payload: dict) -> Response:
```

Comment:
@stephenfuqua EdFiBasicAPIClient is used for change query tests and I think is meant to be used for read-only API calls.

Reply:
It is also used by the login process, I saw the code entering line 94 below outside of the login process as well - which is why I had to add that explicit check that the 401 did not occur on the `/token` endpoint. Because of the way that the old function was written, I had a difficult time seeing the actual error until I had refactored.